src/operations/estimated_document_count.ts

```diff
@@ -36,9 +36,8 @@ export class EstimatedDocumentCountOperation extends CommandOperation<number> {
     if (maxWireVersion(server) < 12) {
       return this.executeLegacy(server, session, callback);
     }
```

Comment:
Can we 

1. get rid of all wire version checks?  We're using the `count` command all the time now.
2. Get rid of `executeLegacy`?

Reply:
Nice catch - I've updated per the suggestions.